rust/src/main.rs

```diff
@@ -102,18 +102,16 @@ struct Cli {
 
 fn main() {
     let cli = Cli::parse();
-    let debug = cli.debug || BooleanKey("debug", false).get_value();
-    let trace = cli.trace || BooleanKey("trace", false).get_value();
-    let log = Logger::create(&cli.output, debug, trace);
+    let debug = cli.debug || boolean_key("debug").unwrap_or(false);
+    let trace = cli.trace || boolean_key("trace").unwrap_or(false);
```

Comment:
This new way to read envs or config file seem not to work. I tested manually by exporting the following env:

```
SE_TRACE=true
```

... and the executing selenium-manager as usual:

```
cargo run -- --browser chrome
```

When this feature works, the `TRACE` logs should be shown. But now only `INFO` (i.e., the default) are shown. 

The config file (`~/.cache/selenium/selenium-manager-config.toml`) should also work to provide extra configuration.

For more info, see #11810.